CMakeLists.txt

```diff
@@ -94,7 +95,7 @@ enable_testing()
 
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
 include(cpplint)
-include(glog)
+#include(glog)
```

Comment:
glog might be needed stil for host code?

Reply:
I have replace all. And log.h is ok for gpu, then it's also for cpu.